app/Jobs/FetchTwitterAvatar.php

```diff
@@ -36,6 +36,17 @@ public function handle()
             $details = app(TwitterOAuth::class)->get('users/show', ['screen_name' => $this->friend->username]);
             $url = str_replace('_normal', '', $details->profile_image_url_https);
 
+            $this->friend->name = $details->name;
```

Comment:
Since this is no longer just about profile image, leaving the profile pic url as `$url` is confusing. Why don't we A) rename it to `$avatar_url`, and B) move it (line 37) down just above line 50?

Also, why not move the description line (47) up to be just below the name line (39)?

That way this reads:

1. get details
2. update clean details: name, description, and location
3. update conditional details: url and url_display
4. save
5. get profile pic